lib/sub-process.ts

```diff
@@ -12,7 +12,7 @@ export function execute(command, args, options): Promise<string> {
     spawnOptions.cwd = options.cwd;
   }
   if (args) {
-    args = quoteAll(args, spawnOptions);
+    args = quoteAll(args, { flagProtection: false });
```

Comment:
`shell: true` was [deprecated](https://nodejs.org/docs/latest-v24.x/api/deprecations.html#dep0190-passing-args-to-nodechild_process-execfilespawn-with-shell-option-true).